modules/vhacd/register_types.cpp

```diff
@@ -32,24 +32,33 @@
 
 #include "scene/resources/mesh.h"
 
+#define ENABLE_VHACD_IMPLEMENTATION 1
 #include "thirdparty/vhacd/public/VHACD.h"
```

Comment:
Since there's only one file, I would remove the `public/` folder and just have `thirdparty/vhacd/VHACD.h`.

Also, remember to update `thirdparty/README.md` with information on what commit is currently included, and what file(s) to copy on updates.

We also had a bunch of patch files which may no longer be relevant, if so they should be removed too.